include/boost/static_string/static_string.hpp

```diff
@@ -3751,6 +3763,24 @@ class basic_static_string
     size_type n,
     value_type c);
 
+  /**
+      Resize the string and overwrite its contents.
+
+      Resizes the string to contain `n` characters, and uses the
```

Comment:
I'm not sure that is useful, the size passed here doesn't really matter, does it? Maybe a `write_and_resize(op)` is more useful? Or is there any realistic use case for this param being less than N?

Reply:
That's how the function is specified for std::string.

For static_string specifically the only thing the passed size does is throw if it's too big.